entities/person.rb

```diff
@@ -35,16 +35,3 @@ def of_age?
     @age >= 18
   end
 end
-
-# testing
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/201259104-907b0f1a-6eb2-447c-ae97-9f2f414e0f67.png)

- [ ] You have added a LICENSE file in your project :blush: and i can see a link in readme file for the licnese file :+1: but the link is broken :fearful: Kindly fix this link to make your readme descriptive :blush:

Reply:
Thank you for posting this out ! It's the first time I create a project with the license template and forgot to update the readme link value